### PR TITLE
[T2A4][F12-A2]Aditya Hendraatmaja

### PR DESCRIPTION
**Pull request by ahendraatmaja**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -450,8 +450,8 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
-        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
-        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
+        final Set<String> keywords = extractKeywordsFromFindPersonArgsInLowerCase(commandArgs);
+        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyCaseInsensitiveKeyword(keywords);
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/AddressBook.java:454]: Glad to see that conscious effort to make the code readable =).
Though for `getPersonsWithNameContainingAnyCaseInsensitiveKeyword` can you think of any other way to shorten the method name yet retain its meaning?
Do you think that `getPersonsWithNameContainingAnyCaseKeyword` conveys the same idea?

Furthermore method comments should also be updated.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -472,8 +472,8 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      * @param findPersonCommandArgs full command args string for the find persons command
      * @return set of keywords as specified by args
      */
-    private static Set<String> extractKeywordsFromFindPersonArgs(String findPersonCommandArgs) {
-        return new HashSet<>(splitByWhitespace(findPersonCommandArgs.trim()));
+    private static Set<String> extractKeywordsFromFindPersonArgsInLowerCase(String findPersonCommandArgs) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/AddressBook.java:475]: Again do remember to update your method comments.

File: test/expected.txt [hunk 3]
```diff
@@ -184,8 +184,9 @@
 || 0 persons found!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo  Phone Number: 222222  Email: benchoo@nus.edu.sg
```
Comment (brandonyeoxg) [inline test/expected.txt:187]: Glad to see that you ran tests when you implemented/extended a feature.
Shouldn't the user guide be updated too? Recall from our tutorial.

**Review by brandonyeoxg — COMMENTED** [review]
@ahendraatmaja Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/AddressBook.java (hunk 1 above)
Comment (ahendraatmaja) [reply src/seedu/addressbook/AddressBook.java:454]: I was thinking that case keyword would be a little bit still rather vague. Thanks for the input, I do feel the method name is a little bit too long haha

**Pull request closed without merging**